test/browser/spec/ol/layer/MapboxVector.test.js

```diff
@@ -141,7 +141,7 @@ describe('ol/layer/MapboxVector', () => {
               version: 8,
               sources: {
                 'foo': {
-                  url: '/spec/ol/data/{z}-{x}-{y}.vector.pbf',
+                  tiles: ['/spec/ol/data/{z}-{x}-{y}.vector.pbf'],
```

Comment:
This change not only makes the style valid, it also turns this test into a regression test for sources without a `url`.